cfssl_scan.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/cloudflare/cfssl/scan"
+)
+
+var scanUsageText = `cfssl scan -- scan a host for issues
+
+Usage of scan:
+        cfssl scan [-short] [SCANFAMILY] HOST
+        cfssl scan [-short] [-scanner regexp] HOST
```

Comment:
Please put an example here. The documentation is confusing, is SCANFAMILY a regexp? What is regexp matching against? What does [SCANFAMILY]* represent?